config/vufind/config.ini

```diff
@@ -1763,10 +1763,41 @@ hide_holdings[] = "World Wide Web"
 ;
 ; Available options:
 ;    Channels - Display links to channels of content related to record
+;    Bookplate - Display a bookplate image or something similar
 ;    Similar - Similarity based on Solr lookup
 ;    WorldCatSimilar - Similarity based on WorldCat lookup
 related[] = "Similar"
 
+; The following settings are for the the related Bookplate module. They can be
+; enabled here by uncommenting below or from another config file of your choice.
+; To add them to another config file just pass the .ini file name and section
+; where they are found to the related module like so:
+;related[] = "Bookplate:Filename:Section"
+
+; URLs for bookplates if enabled. The following tokens can be used:
+;   %%img%% - image name, partial image name or full URL.
+;   %%thumb%% - thumbnail image name, partial thumbnail image name or full URL.
+; If you index the entire path to bookplate images, you will only need to use
+; the token, e.g. bookplate_full = %%img%%, otherwise you can build URLs similar
+; to the ones below.
+;bookplate_full = https://your-institution.edu/bookplates/%%img%%-full.jpg
+;bookplate_thumb = https://your-institution.edu/bookplates/%%thumb%%-thumb.jpg
+
+; Data field with an array of image titles, parallel with bookplate_img_names_field.
+; Is required for alt text, even if bookplate_display_title is set to false.
+;bookplate_titles_field = "donor"
```

Comment:
@bbusenius, sorry for not being clear, but I think you misunderstood my intent here -- I think `bookplate_titles_field`, `bookplate_images_field`, etc. make perfect sense as names for configuration settings, and I wasn't asking you to change those. I just meant that the example values in your comments would be more realistic if you used `donor_str_mv` and `donor_code_str_mv` instead of `donor` and `donor_code`, because the `_str_mv` suffixes would allow those versions to work with the existing VuFind schema without requiring any modifications. I'd suggest reverting commit 1b9e290ae46e6e2d27999a40e416d25f397b95b5 and just editing the config file.

Again, sorry for putting you to extra work here! I'd make the fix myself, except that I don't seem to have write access to this branch.

Reply:
Ah yes, I took that in a bad direction, LOL. This is fixed now.